src/Features/Core/Portable/CodeCleanup/CodeCleanupLogMessage.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Internal.Log;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CodeCleanup
+{
+    internal static class CodeCleanupLogMessage
+    {
+        public static KeyValueLogMessage Create(DocumentOptionSet docOptions)
+        {
+            return KeyValueLogMessage.Create(LogType.UserAction, m =>
+            {
+                m[nameof(CodeCleanupOptions.CodeCleanupInfoBarShown)] = docOptions.GetOption(CodeCleanupOptions.CodeCleanupInfoBarShown);
```

Comment:
you don't need CodeCleanupInfoBarShown since it is runtime only optino

Reply:
Removed it.